### DISCUSSION TITLE
Foros de preguntas y respuestas

### DISCUSSION THREAD

**Discussion by jolufe** [chamilo/chamilo-lms#5285] [General]
Hola ¡Gracias por el nuevo foro!

En Moodle hay un tipo de foro llamado "de preguntas y respuestas" en el que para ver las entradas anteriores es obligatorio publicar primero la entrada propia ¿Hay algo así en Chamilo? Gracias.

Chamilo 1.11.18
Windows 10

![image](https://github.com/chamilo/chamilo-lms/assets/40075497/2fa2eba5-e4ab-4351-ba54-bb3cddcd3581)

**Comment by chatonito** [comment]
Hola Jolife
en chamilo no hay esa opción, porque se intenta fomentar la discusión de temas académicos principalmente, pero puedes crear un foro MODERADO donde los estudiantes pueden preguntar sobre algo y darle una respuesta, ya que existe un botón que dice "responder a este mensaje"; la moderación la puede hacer el docente y no se verá el comentario hasta que se active el botón de "corregido". 
espero haber ayudado, saludos de Morelia Michoacán México
atte
dr agustin rico guzman
webmaster Enseñanza Médica CAR

**Reply by jolufe** [reply]
Muchas gracias por la respuesta. En realidad, la intención de los foros de preguntas y respuestas no es tanto para que responda el docente sino para que se establezca un diálogo entre los participantes a partir de la primera entrada de cada uno. La idea es que mi primera entrada, en respuesta a alguna consigna del docente, no se vea influenciada por lo que respondieron mis compañeros antes que yo, por eso no puedo verlas hasta media hora después de mi primera publicación.

Entiendo la idea del foro moderado, pero me parece que la opción de Moodle es también interesante, como para tenerla en cuenta.

Gracias nuevamente.
Un cordial saludo desde Córdoba, Argentina.
José Luis.

**Reply by contidos** [reply]
Hola Jolufe, para las propuestas de nuevas funcionalidades o ideas, podeis hacerlas llegar al equipo de desarrollo desde la sección de Issues y explicar lo que se desea conseguir. El equipo de desarrollo lo podrá considerar y etiquetar como propuesta de mejora para futuras versiones. No todas las propuestas se podrán aplicar pero el equipo recibe siempre con mucho agrado ideas de los usuarios que sois los que estais en el día a día de la operativa. Así que si crees que algo puede ser una mejora, no dudes en proponerlo dando los máximos detalles posibles.

**Reply by jolufe** [reply]
Hola Noa. Gracias por la sugerencia. Voy a proponérselo a los desarrolladores.
Un cordial saludo.
